gnss_lib_py/utils/visualizations.py

```diff
@@ -579,6 +549,16 @@ def _get_label(inputs):
     if not isinstance(inputs,dict):
         raise TypeError("_get_label input must be dictionary.")
 
+    # handle units specially.
+    units = {"m","km",
+             "deg","rad",
+             "sec","s","hr","min",
+             "mps",
+             }
+    unit_replacements = {
+                         "mps" : "m/s",
```

Comment:
If this is supposed to be an exhaustive list, I would also recommend adding kmph, mph and deg/s, rad/s for angular velocity. Same for acceleration terms that you might encounter in a filter

Reply:
kmph : km/hr
mph : miles/hr
degps : deg/s
radps : rad/s